src/ens/storage.h

```diff
@@ -0,0 +1,72 @@
+#ifndef CONTACT_STORAGE_H
+#define CONTACT_STORAGE_H
+
+#include "../contacts.h"  // Requires contact_t in contacts.h!
+#include "../covid_types.h"
+#include "sequencenumber.h"
+
+typedef uint16_t storage_id_t;
+
+// TODO lome: own datatype for storage
+typedef struct record {
+    record_sequence_number_t sn;  // TODO: Convert Sequence Number
+    uint32_t timestamp;           // TODO: Seconds from january first 2000 (UTC+0)
+    rssi_t rssi;                  // TODO: Check correct
+    rolling_proximity_identifier_t rolling_proximity_identifier;
+    associated_encrypted_metadata_t associated_encrypted_metadata;
+} record_t;
+
+typedef struct stored_contacts_information {
+    record_sequence_number_t oldest_contact;
+    record_sequence_number_t count;
```

Comment:
Count should not be a sequence number! (32 bits should be enough I guess)
```suggestion
    uint32_t count;
```